api/v1alpha1/condition_consts.go

```diff
@@ -37,6 +37,14 @@ const (
 	VirtualMachineImageNotFoundReason = "VirtualMachineImageNotFound"
 )
 
+const (
+	// DeployedCondition exposes the status of the VirtualMachine deployment.
```

Comment:
I think the `VirtualMachine deployment` reads as `VirtualMachineDeployment`. Could you update this to reflect that we are talking about the VM being deployed on the infrastructure provider?

Reply:
will do. Thanks for taking a look